apigenerator/resources/1 - Project/1 - BaseProject/Project/src/e_Infra/b_Builders/DomainBuilder.py

```
@@ -211,7 +211,9 @@ def auto_fill_guid_in_request_body(declarative_meta, dictionary):
     for column in ins.tables[0].columns:
         if column.primary_key:
             if column.name not in dictionary:
-                if str(column.type) in auto_fill_guid_allowed_types:
+                if str(column.type) == 'UUID':
+                    dictionary[column.name] = generate_uuidv7()
```

Ruff (0.11.9)

`generate_uuidv7` may be undefined, or defined from star imports

(F405)